BitPlanner/Scenes/CraftPage.cs

```diff
@@ -132,6 +133,61 @@ public void CopyTreesAsText()
         DisplayServer.ClipboardSet(result.ToString().Trim());
     }
 
+    public void CopyTreesAsCSV()
+	{
+	var result = new StringBuilder();
+	var separator = new string('-', 64);
+
+	result.AppendLine("Item Name,Generic Name,Profession/Skill,Quantity,In Stock");
+
+		foreach (var tab in _recipeTabs.GetChildren().Cast<RecipeTab>())
+		{
+		var root = tab.GetTreeRoot();
+		TraverseAndAppendCSV(root, result);
+		result.AppendLine(separator); // Separera olika träd
+		}
+
+	DisplayServer.ClipboardSet(result.ToString().Trim());
+	}
+
+	private void TraverseAndAppendCSV(TreeItem item, StringBuilder csv)
+	{
+		var id = item.GetMetadata(0).AsUInt64();
+		var craftingItem = _data.CraftingItems[id];
+
+		// Get quantity
+		var quantity = item.GetMetadata(2).AsInt32Array();
+		var minQuantity = quantity[0];
+		var maxQuantity = quantity[1];
+
+		var quantityString = RecipeTab.GetQuantityString((uint)minQuantity, maxQuantity >= 0 ? (uint)maxQuantity : 0u);
```

Comment:
```suggestion
		var quantity = item.GetMetadata(2).AsGodotArray();
		var minQuantity = quantity[0].AsUInt32();
		var maxQuantity = quantity[1].AsUInt32();

		var quantityString = RecipeTab.GetQuantityString(minQuantity, maxQuantity);
```

Quantities are always >= 0 here, it's safe to use uint.